ldk/javascript/examples/self-test-loop/src/tests/network/index.ts

```diff
@@ -97,65 +99,73 @@ export const testHttpRequestBlock = (): Promise<boolean> =>
     }
   });
 
+const browserFake = async (
+  socket: network.Socket,
+  msg: string | Uint8Array,
+  reject: (arg0: unknown) => void,
+): Promise<void> => {
+  if (msg && typeof msg === 'string') {
+    console.info(`Received message: ${JSON.stringify(msg)}`);
+    const re = /[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}/g;
+    const [callId] = msg.match(re);
+    try {
+      await socket.writeMessage(
+        `{ "type": "OpenWindowReturn", "version": 0, "callId": "${callId}", "return": { "windowId": ${windowId}, "err": "" }}`,
+      );
+    } catch (err) {
+      console.log('failed to write message ', err);
```

Comment:
Some console logs start capitalized and others are all lowercase, should we make this consistently capitalized?